src/meta/app/src/schema/catalog.rs

```diff
@@ -28,3 +32,49 @@ impl Display for CatalogType {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct CatalogMeta {
+    pub catalog_type: CatalogType,
+    pub options: BTreeMap<String, String>,
+    pub created_on: DateTime<Utc>,
+}
+
+impl Display for CatalogMeta {
```

Comment:
Why do we need to implement `Display`? And it's content seems the same with `Debug`.

Reply:
To align with other structs in `meta`

Reply:
cc @drmingdrmer for comments.

I prefer to remove not needed functions instead of `align`.

Reply:
Agree. Add only what you need.